src/patternfly/components/Nav/nav.scss

```diff
@@ -203,6 +207,34 @@
   }
 
   position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  overflow: hidden;
```

Comment:
This display/flex-wrap/overflow block looks OK but I'm wondering if it's going to introduce a bug with vertical nav (particularly `overflow: hidden`), and it's only needed on horizontal/tertiary nav.

Reply:
https://github.com/patternfly/patternfly/issues/2973